src/index.html

```diff
@@ -3,11 +3,130 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>Miami Condo Kings</title>
     <link rel="stylesheet" href="./styles/main.css">
+    <link rel="icon" href="images/favicon.png">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,400i,500,500i,700,700i&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Miami</h1>
     <script type="text/javascript" src="scripts/main.js"></script>
+    <header class="header">
+      <div class="header__top">
+        <div class="header__phone">+1 234 555-55-55</div>
+        <div class="logo header__logo"></div>
+        <div class="header__action"><a href="#" class="header__link">Call to order</a></div>
+      </div>
+      <div class="header__title">Miami<br>Condo Kings</div>
+      <nav class="nav header__nav">
+        <ul class="nav__list">
+          <li class="nav__item nav__item_active">Home</li>
+          <li class="nav__item"><a href="#" class="nav__link">About us</a></li>
+          <li class="nav__item"><a href="#" class="nav__link">Gallery</a></li>
+          <li class="nav__item"><a href="#" class="nav__link">Consultation</a></li>
+          <li class="nav__item"><a href="#" class="nav__link">Testimonials</a></li>
+          <li class="nav__item"><a href="#" class="nav__link">Contacts</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <section class="section">
+      <div class="info">
+        <h1 class="info__title">Miami best <br> Real estate <a class="info__link" href="#">here</a></h1>
+        <div class="info__text">
+          <p class="info__heading">Over 50 years experience in love.</p>
+          <p>Mazel and VA Tsukkerman in his "Analysis of musical works." <br>Pointillism, which originated in the music of the early twentieth <br> century, microforms, found a distant historical.</p>
+          <p>The market leader in real estate since 2003. Awards Real Estate <br> Company, 2010, 2011, 2013. More than 200 satisfied customers <br>premium segment. We focus only on new buildings and high-level <br> doskanalno know this area better than other specialists.<br>Working directly with developers - so have more useful information<br> about the profitable deals.</p>
```

Comment:
переноси длинные теги
```suggestion
          <p>
            The market leader in real estate since 2003. Awards Real Estate <br> Company, 2010, 2011, 2013. More than 200 satisfied customers <br>premium segment. We focus only on new buildings and high-level <br> doskanalno know this area better than other specialists.<br>Working directly with developers - so have more useful information<br> about the profitable deals.
          </p>
```
хотя бы так
я бы рекомендовал тебе избегать `<br />`, вместо этого оборачивать отдельные части текста в div